README.md

```diff
@@ -33,14 +33,28 @@ After installing docker, build the docker image:
 docker build -t bot-render . --no-cache=true
 ```
 
-Start a container with the built image:
+## Running the container
+
+There are two ways to run the container locally:
+1. With [Jessie Frazelle seccomp profile](https://github.com/jessfraz/dotfiles/blob/master/etc/docker/seccomp/chrome.json)
+2. With `--cap-add=SYS_ADMIN`
+
+If you do not use one of the options above, you will get `ECONNREFUSED` errors when trying to access the container as noted in issues [2](https://github.com/samuelli/bot-render/issues/2) and [3](https://github.com/samuelli/bot-render/issues/3).
```

Comment:
Depends on your underlying kernel. I can see where on a more recent Ubuntu or Arch where this is going to happen (and I can dupe the issue based on what was reported in those tickets).